content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Datadog FIPS Compliance
+kind: Guide
+disable_toc: false
+further_reading:
+- link: "agent/proxy"
+  tag: "Documentation"
+  text: "Agent Proxy Configuration"
+algolia:
+  rank: 80
+  tags: ["fips", "fips proxy", "compliance"]
+---
+
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+<div class="alert alert-warning">The Datadog FIPS proxy is only available in the US1-FED region.</a></div>
+{{< /site-region >}}
+
+The Datadog FIPS proxy ensures that communication between the Datadog Agent and Datadog uses FIPS-compliant encryption.
+
+The Datadog FIPS proxy is a separately-distributed component that is deployed on the same host as the Datadog Agent. The proxy acts as an intermediary between the Agent and Datadog. The Agent communicates with the Datadog FIPS proxy, which encrypts payloads using a FIPS 140-2 validated cryptographic module and relays the payloads to Datadog.
+
+## Supported platforms and limitations
+
+The Datadog FIPS proxy's compliance is based on its use of the FIPS 140-2 validated [Cryptographic Module - Certificate #4024][1]. See the related [security policy][2] for information around validated operating environments and restrictions. It is ultimately the end user's responsibility to ensure operating environment compliance with the security policy and wider FIPS guidance.
```

Comment:
This second sentence around user/customer's responsibility needs to be highlighted and placed prominently